Aptoide/aptoide-client-v8
# Bugfix/asv 982 - Fix TC and PP message

**What does this PR do?**

   Makes the TC and PP message and checkbox to be according to zeplin design. The text is now black and aligned to the left.

**Database changed?**

 No

**Where should the reviewer start?**

- [x] layout/fragment_login_sign_up_credentials.xml
- [x] layout-land/fragment_login_sign_up_credentials.xml

**How should this be manually tested?**

  Check the screen on Login/Signup, and see if the checkbox and TC and PP message is black and aligned to left. Rotate the screen and check the same.

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/ASV-982


**Code Review Checklist**
